Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -842,27 +847,39 @@ public int GetGoldAmount()
         /// <returns>Amount remaining to be paid if not enough funds.</returns>
         public int DeductGoldAmount(int amount)
         {
-            if (amount <= goldPieces) {
+            if (amount <= goldPieces)
+            {
                 goldPieces -= amount;
-            } else {
+            }
+            else
+            {
```

Comment:
It's going to expand a lot of compressed blocks like this, since it's my coding style to do this for small blocks of code as I find it more readable to not use extra lines in these situations.

If the team decision is to have them expanded, that is perfectly fine, I'm just raising it so a decision can be taken. For all I know my style has been annoying everyone. Anyway we should go into it eyes wide open.